src/convert/streams.ts

```diff
@@ -150,9 +151,22 @@ export class StandardWriter extends ComponentWriter {
       try {
         const toResolve: string[] = [];
         const writeTasks = chunk.writeInfos.map((info: WriteInfo) => {
-          const fullDest = isAbsolute(info.output)
+          let fullDest = isAbsolute(info.output)
             ? info.output
             : join(this.rootDestination, info.output);
+          if (!fs.fileExistsSync(fullDest)) {
+            for (const ignoredPath of this.forceIgnoredPaths) {
+              if (
+                dirname(ignoredPath).includes(dirname(fullDest)) &&
+                basename(ignoredPath).includes(basename(fullDest))
+              ) {
+                fullDest = ignoredPath;
```

Comment:
Can we just return here? I think this code can be simplified a bit.

Reply:
Good point! Updated. Thanks Jag!